packages/runtime/test/lib/testUtils.ts

```diff
@@ -315,6 +336,15 @@ export async function sendMessageWithRequest(
     return sendMessageResult.value as MessageDTO & { content: RequestJSON };
 }
 
+export async function createRequest(
```

Comment:
You don't save anything by introducing a helper function for a one-line function